x-pack/plugins/security_solution/public/flyout/left/tabs/insights_tab.tsx

```diff
@@ -37,22 +32,42 @@ import { CORRELATIONS_TAB_ID, CorrelationsDetails } from '../components/correlat
 const insightsButtons: EuiButtonGroupOptionProps[] = [
   {
     id: ENTITIES_TAB_ID,
-    label: ENTITIES_BUTTON,
+    label: (
+      <FormattedMessage
+        id="xpack.securitySolution.flyout.left.entitiesButtonLabel"
```

Comment:
here I'm not sure if we wanted `insights` back, I'm indifferent as long as they are consistent, wdyt?